composer.json

```diff
@@ -18,13 +18,13 @@
         "php": "^7.1.3",
         "ext-json": "*",
         "dragonmantank/cron-expression": "^2.0",
+        "hmazter/laravel-schedule-list": "^1.2",
         "illuminate/cache": "5.8.*",
         "illuminate/config": "5.8.*",
         "illuminate/console": "5.8.*",
         "illuminate/container": "5.8.*",
         "illuminate/contracts": "5.8.*",
         "illuminate/events": "5.8.*",
-        "vlucas/phpdotenv": "^3.0",
```

Comment:
Why have you removed this buddy?

Reply:
It's moved to being the last entry alphabetically (v).

I ran composer require hmazter/laravel-schedule-list - that automatically moved it.

I can put it back to it's original position?

Reply:
@nunomaduro - I have put it back as it was as I am no longer touching the require object.